katas/es6/language/array-api/fill.js

```diff
@@ -35,4 +35,22 @@ describe('`Array.prototype.fill` can fill up an array with one value', () => {
     assert.deepEqual(arr, [1, 42, 3]);
   });
 
+
+  it('is a mutable method, and will change this object itself', function() {
+    const arr = new Array(3);
+    //// arr.fill('foo')
+    arr.fill(8);
+
+    assert.deepEqual(arr, [8,8,8]);
+  });
+  
+  it('when gets passed an object, it will copy the reference and fill the array with references to that object.', function(){
```

Comment:
```suggestion
  it('when passing an object, it will copy the reference and fill the array with references to that object', function(){
```

just a picky change :)

Reply:
Actually, I just copied the whole sentence from MDN 😅
*  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill

> When the fill method gets passed an object, it will copy the reference and fill the array with references to that object.

I'm not confident at all, but it feel like that it should be a passive voice.